contracts/libs/Actions.sol

```diff
@@ -82,6 +82,8 @@ library Actions {
         address owner;
         // vault id to create
         uint256 vaultId;
+        // vault type, 0 for spread/max loss and 1 for naked margin vault
+        uint256 vaultType;
```

Comment:
are we adding this to the actions library? I thought this might lead to issues and that's why we are having it in the controller as a mapping?